src/applications/vaos/components/ConfirmedAppointment.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Link } from 'react-router';
+import moment from 'moment';
+
+function formatDate(date) {
+  const parsedDate = moment(date, 'MM/DD/YYYY HH:mm:ss');
+
+  if (!parsedDate.isValid()) {
+    return '';
+  }
+
+  return parsedDate.format('MMMM D, YYYY');
+}
+
+function titleCase(str) {
+  return str
+    .toLowerCase()
+    .split(' ')
+    .map(word => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(' ');
+}
+
+function formatTimeFromDate(date) {
+  const parsedDate = moment(date, 'MM/DD/YYYY HH:mm:ss');
+
+  if (!parsedDate.isValid()) {
+    return '';
+  }
+
+  return parsedDate.format('hh:mm a');
+}
+
+export default function ConfirmedAppointment({ appointment }) {
+  const isCommunityCare = appointment.typeOfCareId.startsWith('CC');
```

Comment:
I updated this check in the pending appointment code to be `const isCommunityCare = !!appointment.ccAppointmentRequest;`, since that's what the current app does. I think we could do the same here.

Reply:
Modified, per your suggestion.